meson.build

```diff
@@ -39,13 +39,16 @@ add_project_arguments(
   '-I' + meson.build_root(),
   language: 'cpp', native: true)
 
+# NOTE: -std=gnu11 is required to enable inline assembly.
 add_project_arguments(
-  '-std=c99', '-Wall', '-Werror',
+  '-std=c11', '-Wall', '-Werror',
+  '-std=gnu11', '-Wall', '-Werror',
```

Comment:
I think you haven't resolved this merge conflict correctly.

Reply:
Done.